packages/build-utils/test/fixtures/25-multiple-lock-files-yarn/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "25-multiple-lock-files-yarn",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

```suggestion
  "private": true,
```

Lets remove all the useless fields